From: Mendel Mobach <mendel@mobach.nl>

Add ReiserFS to the list of recognized file systems and move
JFS and XFS defines below the right comment.

---
 agent/mibgroup/hardware/fsys/fsys_mntent.c |    1 +
 agent/mibgroup/hardware/fsys/mnttypes.h    |   13 ++++++-------
 2 files changed, 7 insertions(+), 7 deletions(-)

Index: net-snmp/agent/mibgroup/hardware/fsys/fsys_mntent.c
===================================================================
--- net-snmp.orig/agent/mibgroup/hardware/fsys/fsys_mntent.c
+++ net-snmp/agent/mibgroup/hardware/fsys/fsys_mntent.c
@@ -135,6 +135,7 @@ _fsys_type( char *typename )
               !strcmp(typename, MNTTYPE_GFS2) ||
               !strcmp(typename, MNTTYPE_XFS) ||
               !strcmp(typename, MNTTYPE_JFS) ||
+              !strcmp(typename, MNTTYPE_REISERFS) ||
               !strcmp(typename, MNTTYPE_LOFS))
        return NETSNMP_FS_TYPE_OTHER;
 
Index: net-snmp/agent/mibgroup/hardware/fsys/mnttypes.h
===================================================================
--- net-snmp.orig/agent/mibgroup/hardware/fsys/mnttypes.h
+++ net-snmp/agent/mibgroup/hardware/fsys/mnttypes.h
@@ -114,6 +114,12 @@
 #ifndef MNTTYPE_GFS2
 #define MNTTYPE_GFS2     "gfs2"
 #endif
+#ifndef MNTTYPE_XFS
+#define MNTTYPE_XFS      "xfs"
+#endif
+#ifndef MNTTYPE_JFS
+#define MNTTYPE_JFS      "jfs"
+#endif
 
 /*
  *  File systems to monitor, but not covered by HR-TYPES enumerations
@@ -156,12 +162,5 @@
 #ifndef MNTTYPE_RPCPIPE
 #define MNTTYPE_RPCPIPE  "rpc_pipefs"
 #endif
-#ifndef MNTTYPE_XFS
-#define MNTTYPE_XFS      "xfs"
-#endif
-#ifndef MNTTYPE_JFS
-#define MNTTYPE_JFS      "jfs"
-#endif
-
 
 #endif /* _NETSNMP_FSYS_MNTTYPES_H */
